### PR TITLE
Migrate Athena virtual schema from the virtual-schemas repository

### LINKED ISSUE #1: Migrate `Athena` virtual schema from the virtual-schemas repository

### PR DESCRIPTION
**Pull request by chiaradiamarcelo**
Closes #1.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+[![Code Smells](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Aathena-virtual-schema&metric=code_smells)](https://sonarcloud.io/dashboard?id=com.exasol%3Aathena-virtual-schema)
+[![Coverage](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Aathena-virtual-schema&metric=coverage)](https://sonarcloud.io/dashboard?id=com.exasol%3Aathena-virtual-schema)
+[![Duplicated Lines (%)](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Aathena-virtual-schema&metric=duplicated_lines_density)](https://sonarcloud.io/dashboard?id=com.exasol%3Aathena-virtual-schema)
+[![Lines of Code](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Aathena-virtual-schema&metric=ncloc)](https://sonarcloud.io/dashboard?id=com.exasol%3Aathena-virtual-schema)
+
+# Overview
+
+The **Athena Virtual Schema** provides an abstraction layer that makes an external [DB2](https://www.ibm.com/db2/) accessible from an Exasol database through regular SQL commands. The contents of the external Athena database are mapped to virtual tables which look like and can be queried as any regular Exasol table.
```
Comment (AnastasiiaSergienko) [inline README.md]: Change the link to the Athena :)

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-assembly-plugin</artifactId>
+				<version>3.3.0</version>
+				<configuration>
+					<descriptors>
+						<descriptor>src/assembly/all-dependencies.xml</descriptor>
+					</descriptors>
+					<finalName>virtual-schema-dist-${vscjdbc.version}-db2-${version}</finalName>
```
Comment (AnastasiiaSergienko) [inline pom.xml]:
```suggestion
					<finalName>virtual-schema-dist-${vscjdbc.version}-athena-${version}</finalName>
```

File: doc/changes/changes_1.0.0.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Athena Virtual Schema 1.0.0, released 2021-01-15
+
+Code name: Migration from the virtual-schemas repository
+
+## Features / Enhancements
+
+* 1: Migrated from the virtual-schemas repository
```
Comment (AnastasiiaSergienko) [inline doc/changes/changes_1.0.0.md]:
```suggestion
* #1: Migrated from the virtual-schemas repository 
```

Comment (AnastasiiaSergienko) [reply doc/changes/changes_1.0.0.md]: Otherwise the release droid won't detect a ticket

File: doc/user_guide/athena_user_guide.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+1. In dialog "Add EXACluster JDBC driver" configure the JDBC driver (see below)
+
+You need to specify the following settings when adding the JDBC driver via EXAOperation.
+
+| Parameter | Value                                               |
+|-----------|-----------------------------------------------------|
+| Name      | `ATHENA`                                           |
+| Main      | `com.simba.athena.jdbc.Driver`                  |
+| Prefix    | `jdbc:awsathena:`                                 |
+| Files     | `AthenaJDBC42_<JDBC driver version>.jar`       |
```
Comment (AnastasiiaSergienko) [inline doc/user_guide/athena_user_guide.md]: make it pretty please

**Review by AnastasiiaSergienko — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (chiaradiamarcelo) [reply README.md]: nice catch!

File: doc/user_guide/athena_user_guide.md (hunk 4 above)
Comment (chiaradiamarcelo) [reply doc/user_guide/athena_user_guide.md]: Yes, I wonder if the single quotes are messing this up. I change it locally but the remote version still shows this scrambled.

**Review by AnastasiiaSergienko — APPROVED** [review]

**Pull request merged**